### PR TITLE
dev-python/matplotlib2tikz: Version bump

### PR DESCRIPTION
**Pull request by marbre**

### CODE REVIEW

**Comment by holgersson32644** [comment]
Hi, do you know what is about KEYWORDS="~x86"?

**Comment by marbre** [comment]
@holgersson32644 Never tested it on ~x86, but I see no reason why it shoudn't work on x86.

**Comment by holgersson32644** [comment]
@marbre I don't see any either and suggest just adding the keyword - after all, it's *testing*.
However, maybe a dev want's to drop a comment about that :)

**Comment by amadio** [comment]
Keywords should only be added for tested platforms (e.g. at least it compiles and installs).
https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades

**Comment by holgersson32644** [comment]
@amadio: OK, thanks.
"Important: New packages should be marked as ~arch only upon architectures for which the committing developer has tested."

I’m testing it right now inside a ~x86-container, but that will take a while as my resources are limited on that machine and I have to emerge a bunch of deps (numpy etc) first.

**Comment by holgersson32644** [comment]
@amadio, @mabre: The tests with 0.6.14 and 0.6.15 on ~x86/hardened/glibc were successful with the example from https://github.com/nschloe/matplotlib2tikz.
The output was - beside the version string - identical.

File: dev-python/matplotlib2tikz/matplotlib2tikz-0.6.15.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+DESCRIPTION="Convert matplotlib figures into TikZ/PGFPlots"
+HOMEPAGE="https://github.com/nschloe/matplotlib2tikz"
+SRC_URI="https://github.com/nschloe/matplotlib2tikz/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="test"
+
+RDEPEND="
```
Comment (mgorny) [inline dev-python/matplotlib2tikz/matplotlib2tikz-0.6.15.ebuild]: Missing dep on six.

Comment (marbre) [reply dev-python/matplotlib2tikz/matplotlib2tikz-0.6.15.ebuild]: Added.

**Comment by marbre** [comment]
@holgersson32644 Thanks for testing. Anyway, repoman complains as ImageHash is not available for ~x86. Feel free to send a PR fixing the deps if you like.

**Comment by mgorny** [comment]
@marbre, thanks tested now. As for x86 keywords, I don't mind them being added, as long as someone adds them to all necessary packages ;-).

**Pull request closed without merging**